packages/test-utils/src/utils.ts

```diff
@@ -53,8 +53,8 @@ export const fromBytes = (bytes: Uint8Array): string =>
 
 export const fakePorterUri = 'https://_this_should_crash.com/';
 
-const makeFakeProvider = (timestamp: number, blockNumber: number) => {
-  const block = { timestamp };
+const makeFakeProvider = (timestamp: number, blockNumber: number, blockHash: string) => {
```

Comment:
Not relevant for this PR, but shouldn't the code formatter split this line in several ones?


```suggestion
const makeFakeProvider = (
  timestamp: number,
  blockNumber: number,
  blockHash: string,
) => {
```